### DISCUSSION TITLE
Building baresip for iOS

### DISCUSSION THREAD

**Discussion by kirankumar-ravuri** [baresip/baresip#2540] [General]
Hi All

I followed steps mentioned [here](https://github.com/baresip/baresip-ios) and facing errors while building.

```
make[1]: *** No rule to make target `all'. Stop.
make: *** [libre] Error 2
```

Would someone like to give any inputs?

BRs
Kiran

**Comment by kirankumar-ravuri** [comment]
Hi All

Any help here ?

BRs
Kiran

**Comment by frogblueDFi** [comment]
Hi Kiran!

I gave it up trying the baresip for iOS repo.
Instead cross-compiled everything on my own and linked all the modules statically for iOS and macOS.
openSSL, re, libtiff, spandsp, baresip, ...
Nothing compiles out of the box, it's a pain to guess the correct configure/cmake parameters and export variables.
Good luck! If you need some assist, let me know.

Dietmar

**Reply by kirankumar-ravuri** [reply]
@frogblueDFi That would be great. Please let me know your slackid or send mail to kiranravuri04 at gmail dot com

Thanks
